README.md

```diff
@@ -126,55 +173,12 @@ state of the art models with high quality engineering.
 </tr>
 </table>
 
-### Setting up a development environment
```

Comment:
This section really repeats what's in the installation section.  I copied the most relevant parts over.